### PR TITLE
Fix: Cover repository

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] adds tests for the `IlluminateSpeakerRepository`

### CODE REVIEW

File: tests/Unit/Infrastructure/Persistence/IlluminateSpeakerRepositoryTest.php [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            'save',
+        ]);
+
+        $user
+            ->expects($this->once())
+            ->method('save');
+
+        $repository = new IlluminateSpeakerRepository($this->createUserMock());
+
+        $repository->persist($user);
```
Comment (localheinz) [inline tests/Unit/Infrastructure/Persistence/IlluminateSpeakerRepositoryTest.php]: I am not sure what [`IlluminateSpeakerRepository::persist()`](https://github.com/opencfp/opencfp/blob/70b5aff3fb766b38642a6f2ae3f350871d740fae/classes/Infrastructure/Persistence/IlluminateSpeakerRepository.php#L49-L52) is supposed to return, as the method `Model\User::save()` appears to be a magic method.

Comment (chartjes) [reply tests/Unit/Infrastructure/Persistence/IlluminateSpeakerRepositoryTest.php]: If I recall the pattern correctly, the persist() method should return a boolean value based on whether or not it was able to create a record in the database for $user

Comment (BackEndTea) [reply tests/Unit/Infrastructure/Persistence/IlluminateSpeakerRepositoryTest.php]: yes the [save()](https://laravel.com/api/5.5/Illuminate/Database/Eloquent/Model.html#method_save function returns a boolean, whether it was successful or not.

**Comment by localheinz** [comment]
Thank you, @chartjes!

File: tests/Unit/Infrastructure/Persistence/IlluminateSpeakerRepositoryTest.php (hunk 1 above)
Comment (localheinz) [reply tests/Unit/Infrastructure/Persistence/IlluminateSpeakerRepositoryTest.php]: Thank you, @BackEndTea! 

I don't think we really need to return a `bool` here, what do you think?

**Pull request merged**